cubical/data_handler/ms_tile.py

```diff
@@ -171,6 +171,7 @@ def load_ddfacet_models(self, uvwco, loaded_models, model_source, cluster, imod,
             ndirs = model_source._nclus
             loaded_models[model_source] = {}
             ddfsim.set_model_provider(model_source)
+            log.info("Predicting targetted directions... This may take some time")
```

Comment:
Please amend to targeted.

Reply:
lol good catch. I always seem to make this mistake

Reply:
It's like me and "receipe"... I just cannot train my fingers to do anything else it seems...